tq/ssh.go

```diff
@@ -196,8 +198,7 @@ func (a *SSHAdapter) download(t *Transfer, cb ProgressCallback) error {
 }
 
 // download starts a download. dlFile is expected to be an existing file open in RW mode
-func (a *SSHAdapter) downloadOne(t *Transfer, cb ProgressCallback, dlFile *os.File) error {
-	conn := a.transfer.Connection(0)
+func (a *SSHAdapter) downloadOne(t *Transfer, conn *ssh.PktlineConnection, cb ProgressCallback, dlFile *os.File) error {
```

Comment:
What about changing this function signature to `doDownload(t *Transfer, conn *ssh.PktlineConnection, f *os.File, cb ProgressCallback)` to mirror the `doUpload()` function?  (The comment should become `f is expected ...` if this change was made.)  I feel like that might be nice because they serve such similar purposes, are both internal to this source file, and because I like having the callback parameter at the end of the signature, I guess because it seems less important to me than the file handle.

Reply:
We can certainly do that.